src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -613,5 +616,11 @@ data class IndividualUser(
   private fun isMember(organizationId: OrganizationId?) =
       organizationId != null && organizationId in organizationRoles
 
+  /** Returns true if one of the user's global roles allows them to read an organization. */
+  private fun isGlobalReader(organizationId: OrganizationId) =
+      isSuperAdmin() ||
```

Comment:
Should an accelerator admin also be able to "globally read"?

Reply:
My thinking here (in the next part of this expression) is that accelerator admins can read orgs that have the Accelerator internal tag. So it takes a super-admin to update an organization's internal tags, and after that the non-super admins and others can read the organization. Admittedly this is just my own intuition about how it should work, though.

Reply:
I see, I think that makes sense.